docs/functions.md

```diff
@@ -0,0 +1,73 @@
+# Function Introduction
+
+## Relating with VCS tools
+
+Cyclone can relate with various common VCS tools, such as git, svn, etc. After OAuth, it can pull codes from repository and create webhook. Whenever the user commit, submit a pull request or release a version to the repository, the webhook will trigger the CI/CD workflow. 
```

Comment:
I think it should be "Cyclone has been integrated with a variety of VCS tools", relate is not suitable.